pkg/command/flag/type.go

```diff
@@ -16,6 +18,97 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+// Label describes modification applied to object's labels.
+// It takes a form of csv string slice, where values can have the following formats:
+// k=v - format means that label should be added.
+// k-  - format means that label should be removed.
+// Because of that, the '=' is reserved and cannot be a part of the label.
+type Label struct {
+	add    map[string]string
+	remove []string
+}
+
+var _ flag.Value = (*Label)(nil)
+
+func (l *Label) String() string {
+	var parts []string
+	for k, v := range l.add {
+		parts = append(parts, k+"="+v)
+	}
+	for _, k := range l.remove {
+		parts = append(parts, k+"-")
+	}
+	return strings.Join(parts, ",")
+}
+
+// Set implements pflag.Value.
+func (l *Label) Set(s string) error {
+	if s == "" {
+		return nil
+	}
+	stringReader := strings.NewReader(s)
+	csvReader := csv.NewReader(stringReader)
+	spec, err := csvReader.Read()
+	if err != nil {
+		return fmt.Errorf("parse csv label list: %w", err)
+	}
+	// Taken from https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/label/label.go#L411
+	add := map[string]string{}
```

Comment:
Why not to initialize l.add here and work on that map instead of creating new var and just assign it to l.add at the end
```
l.add = make(map[string]string)
```

?

Reply:
OK, let's leave it as is.